database/console/test_utils.go

```diff
@@ -0,0 +1,112 @@
+package console
+
+import (
+	contractsorm "github.com/goravel/framework/contracts/database/orm"
+	"github.com/goravel/framework/support/file"
+)
+
+func createMigrations(driver contractsorm.Driver) {
+	switch driver {
+	case contractsorm.DriverPostgres:
+		createPostgresMigrations()
+	case contractsorm.DriverMysql:
+		createMysqlMigrations()
+	case contractsorm.DriverSqlserver:
+		createSqlserverMigrations()
+	case contractsorm.DriverSqlite:
+		createSqliteMigrations()
+	}
+}
+
+func createMysqlMigrations() {
+	err := file.Create("database/migrations/20230311160527_create_agents_table.up.sql",
```

Comment:
Can't we use newly implemented migration feature here?

Reply:
Can't for now, will optimize it in another PR.